content/en/docs/reference/deprecation-policy.md

```diff
@@ -36,6 +36,7 @@ includes:
 
    * REST resources (aka API objects)
    * Fields of REST resources
+   * Annotations on REST resources, including "beta" annotations
```

Comment:
What about alpha annotations?

Reply:
concretely, there's a current effort to remove/promote the alpha sysctl annotations to actual fields - https://github.com/kubernetes/kubernetes/pull/63717

Reply:
Didn't we decide to break compatibility for alpha fields and annotations?

https://github.com/kubernetes/community/blob/master/contributors/devel/api_changes.md#adding-unstable-features-to-stable-versions

Reply:
> Didn't we decide to break compatibility for alpha fields and annotations?

Yes, just wanted to make sure this wasn't worded in a way that seemed to prohibit that

Reply:
Nope, I wanted to be clear that this is beta.  Do you want it to be more clear?

Reply:
> Do you want it to be more clear?

I think so. Something like `Non-alpha annotations on REST resources (including "beta" annotations)`